benchmarks/pom.xml

```diff
@@ -120,11 +120,6 @@
        https://github.com/charithe/kafka-junit -->
       <version>4.2.10</version>
```

Comment:
note we have kafka-junit here in the benchmark project, but probably we can migrate that to testcontainers at a low priority